packages/kit/src/runtime/server/index.js

```diff
@@ -114,6 +115,8 @@ export async function respond(request, options, state) {
 		transformPage: default_transform
 	};
 
+	// TODO match route before calling handle?
```

Comment:
probably don't need to add this TODO so that we don't need to remember to remove it in https://github.com/sveltejs/kit/pull/4344

Reply:
#4344 already removes it, it'll probably be an easier merge if we leave it in place